app/src/main/java/com/babylon/wallet/android/presentation/status/dapp/DappInteractionDialog.kt

```diff
@@ -32,6 +32,10 @@ fun DappInteractionDialog(
 ) {
     val state by viewModel.state.collectAsStateWithLifecycle()
 
+    LaunchedEffect(Unit) {
+        viewModel.onShow()
```

Comment:
can't we just call this in a dialog view model init instead of setting up the effect?